fearless/Modules/Pincode/ScreenAuthorization/ScreenAuthorizationPresenter.swift

```diff
@@ -45,7 +45,7 @@ extension ScreenAuthorizationPresenter: LocalAuthInteractorOutputProtocol {
         }
     }
 
-    func didChangeState(from _: LocalAuthInteractor.LocalAuthState) {}
+    func didChangeState(to _: LocalAuthInteractor.LocalAuthState) {}
```

Comment:
unused parameter

Reply:
yes, this is LocalAuthInteractorOutputProtocol